drivers/sensor/apds9960/apds9960.c

```diff
@@ -365,9 +370,7 @@ static int apds9960_init_interrupt(struct device *dev)
 	drv_data->gpio_pin = config->gpio_pin;
 
 	gpio_pin_configure(drv_data->gpio, config->gpio_pin,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_EDGE |
-			   GPIO_INT_ACTIVE_LOW | GPIO_INT_DEBOUNCE |
-			   GPIO_PUD_PULL_UP);
+			   config->gpio_flags);
```

Comment:
I think this should be `GPIO_INPUT | config->gpio_flags` as the devicetree flags don't specify direction.